k8s-install/scripts/expected_10-calico.conf.alternate

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "alternate",
+    "type": "calico",
+    "etcd_endpoints": "",
+    "etcd_key_file": "",
```

Comment:
Does the use case that you're addressing require substitution of alternate `__ETCD_*__` variables here?  If so, if would be good to test that with a case where the substitution is not `""`.

Reply:
I'm only really testing that _some_ values get substituted here; the substitution code is the same whatever the source of the data since the data gets written out to the temporary file before being processed.  I think etcd substitutions can be safely tested orthogonally to my change.